.ci/install_qemu.sh

```diff
@@ -61,7 +64,14 @@ install_packaged_qemu() {
 }
 
 clone_qemu_repo() {
-	git clone --branch "${CURRENT_QEMU_BRANCH}" --single-branch --depth 1 --shallow-submodules "${QEMU_REPO_URL}" "${GOPATH}/src/${QEMU_REPO}"
+	# check if git is capable of shadow cloning
+        git_shadow_clone=$(check_git_version "${GIT_SHADOW_VERSION}")
+
+	if [ "$git_shadow_clone" == "true" ]; then
+		git clone --branch "${CURRENT_QEMU_BRANCH}" --single-branch --depth 1 --shallow-submodules "${QEMU_REPO_URL}" "${GOPATH}/src/${QEMU_REPO}"
+	else
+		git clone --branch "${CURRENT_QEMU_BRANCH}" --single-branch "${QEMU_REPO_URL}" "${GOPATH}/src/${QEMU_REPO}"
```

Comment:
I think you can still have `--depth 1` here ? It is just `--shallow-submodules` iirc that was introduced at 2.9.0.

Reply:
yep, you're right. ;)
I'm doing a little investigation here, seeing from there [official documentation](https://git-scm.com/docs/git-clone/), the oldest version which could be traced back is 2.3.0, the `--depth` option was there. 
So we may have `--depth 1` there.